src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -110,6 +111,17 @@ public void setBrakeMode() {
     rearRightSparkMax.setIdleMode(IdleMode.kBrake);
   }
 
+	@Override
+    public void initSendable(SendableBuilder builder) {
+  	    builder.setSmartDashboardType(getName());
+  	    builder.addDoubleProperty("FrontLeft", () -> { return frontLeftSparkMax.get();} , null);
+  	    builder.addDoubleProperty("FrontRight", () -> { return frontRightSparkMax.get();} , null);
+  	    builder.addDoubleProperty("RearLeft", () -> { return rearLeftSparkMax.get();} , null);  	    
+  	    builder.addDoubleProperty("RearRight", () -> { return rearRightSparkMax.get();} , null);
+
```

Comment:
I think these are the motor speeds not the encoder counts.  
I'd rather see getAveragePosition() and getAverageDistanceMeters() on the dashboard.
I've also looked at the conversion in getAverageDistanceMeters, and I don't think that's right either.  I'll have a patch shortly.